benchmarks/Makefile

```diff
@@ -1,17 +1,34 @@
+# Get the current ISO8601 date time format but remove + and :
+$(shell date -I'minutes'| cut -d+ -f1 | sed 's/:/-/'> ./current)
```

Comment:
I don't like how this is done... Is there a better way of getting this variable in without having to store it in a temp file? Does it really matter?